_haikus/sleepin.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: sleepin
+author: jhotchkiss06
+---
+
+sleep is for the weak
```

Comment:
Please add a `<br>` tag or two spaces `  ` after each line to force a line break.